src/test/java/net/codejava/MvcControllerTest.java

```diff
@@ -0,0 +1,111 @@
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import net.codejava.MvcController;
+import net.codejava.User;
+import org.springframework.ui.Model;
+import org.mockito.Mock;
+import static org.mockito.Mockito.*;
+
+import static org.junit.Assert.*;
+import static org.hamcrest.CoreMatchers.*;
+
+import java.util.ArrayList;
+
+@SpringBootTest(classes = net.codejava.TestudoBankApplication.class)
+public class MvcControllerTest {
+
+	/**
+	 * Spring interprets the @Autowired annotation, and the controller is 
+	 * injected before the test methods are run.
+	 **/
+	@Autowired
+	private MvcController controller;
```

Comment:
This creates an actual `MvcController`, meaning your tests are actually making SQL commands. The tests worked locally for you because you used a real customer's credentials. This won't work on anyone else's local application.